README.md

```diff
@@ -24,27 +25,32 @@ A mighty, modern CSS linter that helps you enforce consistent conventions and av
 
 With stylelint, it's easy to start linting your CSS:
 
-1. Install stylelint: `npm install stylelint`.
-2. Choose whether you want to craft your own config or extend a pre-written, shared config.
+1. Decide whether to use either:
```

Comment:
How about: `Decide how you want to use stylelint:`? And the options could start with `as` or `via`?